CMakeLists.txt

```diff
@@ -48,11 +51,15 @@ endif()
 set(RUN_IN_PLACE ${DEFAULT_RUN_IN_PLACE} CACHE BOOL
 	"Run directly in source directory structure")
 
+message(STATUS "*** Will build version ${VERSION_STRING} ***")
+message(STATUS "Compiling client: " ${BUILD_CLIENT})
+message(STATUS "Compiling server: " ${BUILD_SERVER})
+message(STATUS "Compiling unittests: " ${BUILD_UNITTESTS})
+message(STATUS "Compiling benchmarks: " ${BUILD_BENCHMARKS})
+message(STATUS "Generating documentation: " ${BUILD_DOCUMENTATION})
+message(STATUS "Run in place build: " ${RUN_IN_PLACE})
```

Comment:
I'd prefer if this would print the actual names of the variables, i.e.
```cmake
message(STATUS "BUILD_CLIENT: " ${BUILD_CLIENT})
message(STATUS "BUILD_SERVER: " ${BUILD_SERVER})
message(STATUS "BUILD_UNITTESTS: " ${BUILD_UNITTESTS})
message(STATUS "BUILD_BENCHMARKS: " ${BUILD_BENCHMARKS})
message(STATUS "BUILD_DOCUMENTATION: " ${BUILD_DOCUMENTATION})
message(STATUS "RUN_IN_PLACE: " ${RUN_IN_PLACE})
```